<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QVideoPlayerClass</class>
 <widget class="QMainWindow" name="QVideoPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <widget class="QPushButton" name="m_openButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QMySlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>3</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="m_currentTime">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>3</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_totalTime">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>5</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>3</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_playAndStopButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="m_audioVolume">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="m_skipButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>跳过</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMySlider</class>
   <extends>QSlider</extends>
   <header location="global">include/QMySlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../QtWidgetsApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
